test/controllers/reviews_controller_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class ReviewsControllerTest < ActionDispatch::IntegrationTest
+  test "should get new" do
+    get reviews_new_url
+    assert_response :success
+  end
+
+  test "should get create" do
+    get reviews_create_url
+    assert_response :success
+  end
```

Comment:
not able to review this one; not sure to understand it to be honest...

Reply:
nothing to review here, this was automatically generated and shouldn't have been part of this commit